source/_changelogs/4.8.0.md

```diff
@@ -0,0 +1,39 @@
+- Errors thrown within root level `after` hooks now correctly display in the Test Runner's Command Log. Fixes {% issue 2296 %}.
+- We fixed a regression in {% url "4.6.0" changelog-4-6-0 %} where an XHR response without a body would cause Cypress to throw `Cannot read property '__error' of null`. Fixes {% issue 7518 %}.
+- We fixed a regression in {% url "4.6.0" changelog-4-6-0 %} where using {% url "`cy.visit()`" visit %} to an authenticated URL would error with `Request cannot be constructed from a URL that includes credentials: /__cypress/runner/./wasm/mappings.wasm` Fixes {% issue 7481 %}.
+- We now properly load code from the `pluginsFile` or `supportFile` when they are TypeScript files. Fixes {% issue 7459 %}.
+- utf-8 characters now properly display within error code frames. Fixes {% issue 7507 %}.
+- Errors thrown in a {% url "`cy.on('fail', fn)`" catalog-of-events#Cypress-Events %} fail handler now display a stack trace and code frame pointing to the origin of the error. Fixes {% issue 7512 %}.
+- {% url "`.click()`" click %} now properly clicks on wrapped inline elements when the first child element in the parent element has no width or height. Fixes {% issue 7434 %} and {% issue 7012 %}.
+- {% url "`cy.wrap()`" wrap %} now properly respects the `timeout` option. It also better handles situations when passed a promise that never resolves. Fixes {% issue 5980 %}.
+- When the {% url "`experimentalComponentTesting`" experiments#Component-Testing %} is enabled, Cypress will no longer exit with SIGABRT in certain situations. Fixes {% issue 7572 %}.
+- We fixed a regression in {% url "3.8.0" changelog-3-8-0 %} where the "Tests" button in the Test Runner wouldn't take you back to the tests list in all browsers. Fixes {% issue 5948 %}.
+- Using the `S` shortcut during {% url "`.pause()`" pause %} no longer does anything. This prevents the Test Runner from getting into a 'stuck' state. Fixes {% issue 6867 %}.
+
+**Misc:**
+
+- The design of errors and some iconography displayed in the Test Runner's Command Log have been updated. Addresses {% issue 7499 %}, {% issue 7542 %} and {% issue 7563 %}.
+- The commands in the Test Runner's Command Log now display in the same casing as the original command. Addresses {% issue 5773 %}.
+- The navigation links in the Test Runner now display the correct CSS styles when focused. Addresses {% issue 7491 %}.
+- {% url "`Cypress.config()`" config %} now has TypeScript types for the `taskTimeout` option. Addresses {% issue 7531 %}.
+- TypeScript types for {% url "`cy.screenshot()`" screenshot %} options and {% url "`Cypress.log()`" cypress-log %} have been updated to be more accurate. Addresses {% issue 7445 %}.
+- TypeScript types for `Cypress.automation` have been added. Addresses {% issue 7519 %}.
+- We not display a more accurate error message when passing a browser to the `--browser` flag that is not supported by Cypress. Addresses {% issue 6979 %}.
```

Comment:
grammarrrrrrrgh

Reply:
“We not display”, what’s wrong with that 😆